reference/customers.v3.yml

```diff
@@ -248,11 +248,9 @@ paths:
         **Notes**
         For multi-storefront merchants who wants to give:
         * Global access:
-          * The channel listed in `origin_channel_id` needs `allow_global_logins=true` in their channel settings
-          * Leave `channel_ids` empty
-        * Channel restricted access:
-          * The channel listed in `origin_channel_id` needs to have `allow_global_logins=false` in their channel settings
-          *  Either leave `channel_ids` empty to use the current channel or give the list of channels we grant access
+          * Leave `channel_ids` empty. Default storefront channel should have the Channel Customer Setting `allow_global_logins=true`.
```

Comment:
🤔  Do we explain somewhere that channel needs to have `allow_global_logins=true`?
I'm wondering if developers won't understand it as `to give global access, just leave channel_ids empty`

Reply:
Probably there's a gap in the documentation, as we haven't really explained anywhere in this section what a channel with global access is. I think it might be worth explaining further or at least adding a link.

Reply:
I remember recommending to Charles to write an article about this feature since it's quite complex and difficult to summarise just in the API doc.

Reply:
I have added a separate note with the link to the Customer Setting Channel endpoint at least so they know where they can set/read that setting.